thoth/license_solver/package.py

```diff
@@ -63,8 +65,16 @@ def set_version(self, package_version: Optional[Any]) -> None:
 
     def set_license(self, license_name: Tuple[List[str], bool]) -> None:
         """Set type of license."""
+        file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data/license_without_versions.yaml")
```

Comment:
This will make it work also on other platforms (but most probably we do not want to support them anyway):

```suggestion
        file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data", "license_without_versions.yaml")
```